eclass/l10n.eclass

```diff
@@ -20,6 +20,8 @@ case ${EAPI:-0} in
 	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
 esac
 
+inherit strip-linguas
```

Comment:
You want to go the full migration route, not just force people to migrate on EAPI bump?

It feels better to do what we did with eutils and limit the inheritance to certain EAPIs. So, here, we'd inherit only on <= EAPI 7.

(I know eutils is slightly different because it's gone in EAPI 8, but same principle).

Reply:
But that's what we do? In EAPI 8, l10n.eclass will be replaced by plocale.eclass which doesn't inherit strip-linguas.

Reply:
My fault for doing this just before bed. Absolutely, yes.